package.json

```diff
@@ -57,6 +62,8 @@
     "react-native-expo-image-cache": "https://github.com/wcandillon/react-native-expo-image-cache",
     "react-native-fontawesome": "^6.0.0",
     "react-native-side-menu": "^1.1.3",
+    "react-native-web": "^0.11.5",
+    "react-router": "^5.0.1",
     "react-select": "^3.0.4",
     "react-sidebar": "^3.0.2",
     "react-slider": "^0.11.2",
```

Comment:
All the dependencies you added should be "devDependencies" because their are only used to run tests.